docs/Deep-Dives/building-a-banner-rotator-with-an-admin-interface.md

```diff
@@ -40,15 +40,18 @@ Here are the properties we will require:
 
 after adding that into a new Flotiq Content Type you should see the following:
 
-![The Rotator Content Type Definition in Flotiq](images/building-a-banner-rotator/rotator-content-type-definition-flotiq.png){: .center .border .width75}
+![The Rotator Content Type Definition in Flotiq](images/building-a-banner-rotator/rotator-content-type-definition-flotiq.png){: .center .border .width50}
 
 Hit Save and you can start adding new Rotators!
 
 ## 2. Create an API Key
 
 It's always recommended to use a scoped API key - that way access to content is restricted to specific Content Types and even if you add new data models in the future - your data will be safe.In our case - we will need access to 2 Content Types - `Rotator` and `Media`, so let's add a key for that purpose.
 
-![Flotiq scoped API Keys](images/building-a-banner-rotator/flotiq-api-keys.png){: .center .border .width75}
+<div markdown=1 class="inline-flex width75 center">
+<div markdown=1>![](../API/images/api-keys-menu.png)</div>
+<div markdown=1>![](images/building-a-banner-rotator/flotiq-api-keys.png)</div>
+</div>
```

Comment:
http://localhost:4000/docs/Deep-Dives/building-a-banner-rotator-with-an-admin-interface/#2-create-an-api-key 

something bad happend here, probably because of missing brakeline :)

![image](https://github.com/user-attachments/assets/7df46fd5-ffae-471e-a4b0-78efe5fc621f)

Reply:
also I know the next one is not a foult of this mr, but can we also fix this?

http://localhost:4000/docs/Deep-Dives/search-api-deepdive/#results-aggregation

![image](https://github.com/user-attachments/assets/db898fa4-7469-43ab-b629-fde741bca2cf)